lib/symengine.rb

```diff
@@ -28,10 +30,37 @@ def symbols ary_or_string, *params
       end
     end    
     def Function(n)
-      return SymEngine::UndefFunction.new(n)
+      SymEngine::UndefFunction.new(n)
     end
     def evalf(operand, prec=53, real=false)
-        return _evalf(operand, prec, real)
+      _evalf(operand, prec, real)
+    end
+    def lambdify(exp, *syms)
+      eval(SymEngine::Utils::lambdify_code(exp, syms))
+    end
+  end
+  module Utils
+    class << self
+      REPLACEMENTS = { sin: 'Math.sin', cos: 'Math.cos', tan: 'Math.tan',
+                         asin: 'Math.asin', acos: 'Math.acos', atan: 'Math.atan',
+                         sinh: 'Math.sinh', cosh: 'Math.cosh', tanh: 'Math.tanh',
+                         asinh: 'Math.asinh', acosh: 'Math.acosh', atanh: 'Math.atanh', 
+                         pi: 'Math::PI', E: 'Math::E', I: '::Complex::I',
+                         dirichlet_eta: 'SymEngine::Utils::evalf_dirichlet_eta',
+                         zeta: 'SymEngine::Utils::evalf_zeta', gamma: 'Math.gamma' }.map { |from, to| [/(\b#{from}\b)/, to] }.to_h.freeze
+      def evalf_dirichlet_eta(exp)
+        SymEngine::evalf(SymEngine::dirichlet_eta(exp))
+      end
+      def evalf_zeta(exp)
+          SymEngine::evalf(SymEngine::zeta(exp))
+      end
+      def lambdify_code(exp, syms)
+        str = exp.to_s
```

Comment:
It seems this line is unnecessary, you can just do `body = REPLACEMENTS.inject(exp.to_s){...}` below (and note the variable name change: `body` seems more meaningful than `str`, don't you think?..)